test/e2e/storage/persistent_volumes-local.go

```diff
@@ -459,14 +462,21 @@ func createLocalPod(config *localTestConfig, volume *localTestVolume) (*v1.Pod,
 }
 
 // Create corresponding write and read commands
-// to be executed inside containers with local PV attached
+// to be executed via SSH on the node with the local PV
 func createWriteAndReadCmds(testFileDir string, testFile string, writeTestFileContent string) (writeCmd string, readCmd string) {
 	testFilePath := filepath.Join(testFileDir, testFile)
 	writeCmd = fmt.Sprintf("mkdir -p %s; echo %s > %s", testFileDir, writeTestFileContent, testFilePath)
 	readCmd = fmt.Sprintf("cat %s", testFilePath)
 	return writeCmd, readCmd
 }
 
+// Create command to verify that the file doesn't exist
+// to be executed via SSH on the node with the local PV
+func createFileDoesntExistCmd(testFileDir string, testFile string) string {
+	testFilePath := filepath.Join(testFileDir, testFile)
+	return fmt.Sprintf("[ ! -e %s ]", testFilePath)
+}
+
```

Comment:
So in the successful case, the read command has a non-zero return code because the file doesn't exist.  But the `IssueSSH` function validates that the return code is 0, so it returns an error.  Instead of refactoring the `IssueSSH` command, I decided to rework the command we pass in so that it returns a 0 return code if the file doesn't exist.